src/gtfs/read.rs

```diff
@@ -584,6 +604,34 @@ fn manage_comment_from_stop(
     comment_links
 }
 
+fn manage_odt_comment_from_stop_time(
+    comments: &mut CollectionWithId<objects::Comment>,
+    companies: &mut CollectionWithId<objects::Company>,
+    on_demand_transport_comment: Option<String>,
+    company_id: &str,
+) -> Option<String> {
+    if let Some(message) = on_demand_transport_comment {
+        if let Some(company) = companies.get(company_id) {
+            let comment_id = "ODT:".to_string() + &company.id;
```

Comment:
Maybe using `format!` would be more idiomatic.
```suggestion
            let comment_id = `format!("ODT:{}", company.id);
```